### ISSUE TITLE
Pre-trained model available? 

### ISSUE THREAD

**Issue by chihyaoma** [szagoruyko/wide-residual-networks#3]
HI, szagoruyko

Thank you for the excellent code! I am developing a CNN for activity recognition. Our model will largely benefit with a pre-trained model. I am wondering if you will provide the pre-trained model for CIFAR-10 or other datasets like imagenet? 

Thank you!

**Comment by szagoruyko** [comment]
Yes, I will release CIFAR and ImageNet models as soon as I get them trained.

**Comment by chihyaoma** [comment]
Okay. Looking forward to your results!

**Comment by engharat** [comment]
Szagorukyo could you provide the network structure you are using for training on imagenet? What is the depth and width used on that case?
